Source/Frontend/UI/Extensions/DataGridViewDraggable.cs

```diff
@@ -0,0 +1,188 @@
+namespace RTCV.UI
```

Comment:
Can you remove this one from this PR & do a separate PR moving it into  UI/Components/Controls and update the namespace accordingly?